kratos-hydra/package.json

```diff
@@ -22,40 +26,39 @@
     "prepublishOnly": "npm run build"
   },
   "dependencies": {
-    "@ory/client": "^1.1.0",
-    "@ory/elements-markup": "^0.0.1-alpha.28",
-    "@ory/integrations": "^1.1.0",
-    "axios": "^1.2.1",
-    "body-parser": "^1.20.1",
-    "cookie-parser": "^1.4.6",
-    "csurf": "^1.11.0",
-    "express": "^4.18.2",
-    "express-handlebars": "^6.0.6",
-    "express-jwt": "^7.7.7",
-    "express-winston": "^4.2.0",
-    "handlebars-helpers": "^0.10.0",
-    "jwks-rsa": "^3.0.0",
-    "winston": "^3.8.2"
+    "@ory/client": "1.1.23",
+    "@ory/elements-markup": "0.0.1-beta.6",
+    "@ory/integrations": "1.1.0",
+    "axios": "1.2.6",
+    "body-parser": "1.20.1",
+    "cookie-parser": "1.4.6",
+    "csurf": "1.11.0",
+    "express": "4.18.2",
+    "express-handlebars": "6.0.7",
+    "express-jwt": "8.4.0",
+    "express-winston": "4.2.0",
+    "jwks-rsa": "3.0.1",
+    "winston": "3.8.2"
```

Comment:
i see the versioning type has been changed in all the above dependencies from compatbility to equality. not sure what the Ory standard is here.